CONTRIBUTING.md

```diff
@@ -212,3 +212,30 @@ If you were working on a feature branch, do this:
 
 If your feature branch is ready or you are an external contributor with your own
 fork, use the github web interface to open a pull request.
+
+
+# C Source Code Formatting
+
+When editing an existing file, please be polite and first follow the 
+guidelines below, and secondly follow the general rule of keeping to the 
+same formatting conventions as exist in the file.
+
+
+## C Function Declarations
+
+C function declarations should have the type on one line and the 
+function name and parameters on the following line, e.g.:
+
+```
+static int 
+convertCopy(char *from, int fromLen, char *to, int toLen, int term)
+```
+
+This facilitates searching for function definitions by searching for the 
+name at the beginning of the line.
+
+
+## Tabs are 4 spaces
+
+The heading says it all.
```

Comment:
I'm afraid the title doesn't answer the question *when to use* spaces/tabs? Are tabs banned?

Reply:
A side note: the legacy platforms/win32 is using these settings:

8-space tabs stops
2-space indent

I don't want to create massive change of ownership for fixing that...
https://stackoverflow.com/questions/4112410/git-change-styling-whitespace-without-changing-ownership-blame